### PR TITLE
Update README.md - Pycharm instructions not working for files path containing white spaces

### PR DESCRIPTION
**Pull request by Heaford**
Currently, If there is a space in the file path of the python file in PyCharm then an error is generated

Error: Invalid value for "[SRC]...": Path .... does not exist.

It stops the file path at the space.

By enclosing the Arguments in quotes it fixes this. 

Also a small issue in that the instructions for how to access settings were incorrect on the Mac

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
Thanks for your PR! As with #680 and #681, I'm not comfortable merging this without hearing from more people who actually use PyCharm. @Heaford and @gsganden, maybe you could review each other's PRs?

File: README.md [hunk 1]
```diff
@@ -635,21 +635,29 @@ $ where black
 %LocalAppData%\Programs\Python\Python36-32\Scripts\black.exe  # possible location
 ```
 
-3. Open External tools in PyCharm with `File -> Settings -> Tools -> External Tools`.
+3. Open External tools in PyCharm  
+
+  On macOS:
+  
+```PyCharm -> Preferences -> Tools -> External Tools```
```
Comment (gsganden) [inline README.md]: I can confirm that this step is correct.

File: README.md [hunk 2]
```diff
@@ -635,21 +635,29 @@ $ where black
...
 4. Click the + icon to add a new external tool with the following values:
     - Name: Black
     - Description: Black is the uncompromising Python code formatter.
     - Program: <install_location_from_step_2>
-    - Arguments: `$FilePath$`
+    - Arguments: `"$FilePath$"`
 
 5. Format the currently opened file by selecting `Tools -> External Tools -> black`.
-    - Alternatively, you can set a keyboard shortcut by navigating to `Preferences -> Keymap -> External Tools -> External Tools - Black`.
+    - Alternatively, you can set a keyboard shortcut by navigating to `Preferences or Settings -> Keymap -> External Tools -> External Tools - Black`.
```
[the remark this answers is not in the document]
Comment (gsganden) [reply README.md]: Yes, as above the menu path is `PyCharm` -> `Preferences` etc. on Mac. I believe it's `File` -> `Prefereces` etc. on other platforms.

File: README.md [hunk 3]
```diff
@@ -635,21 +635,29 @@ $ where black
...
+  On Windows / Linux / BSD:
+  
+```File -> Settings -> Tools -> External Tools```
 
 4. Click the + icon to add a new external tool with the following values:
     - Name: Black
     - Description: Black is the uncompromising Python code formatter.
     - Program: <install_location_from_step_2>
-    - Arguments: `$FilePath$`
+    - Arguments: `"$FilePath$"`
```
Comment (gsganden) [inline README.md]: I can confirm that running Black inside PyCharm on a file with spaces in its name raises an error without this change and that this change resolves the error.

**Pull request merged**